src/interactions/coulomb.jl

```diff
@@ -115,3 +115,137 @@ end
 @fastmath function potential(::Coulomb, r2, invr2, (coulomb_const, qi, qj))
     (coulomb_const * qi * qj) * √invr2
 end
+
+@doc raw"""
+    CoulombSoftCore(; cutoff, sc_softness, sc_lambda, sc_power, nl_only, lorentz_mixing, weight_14,
+                    coulomb_const, force_units, energy_units)
+
+The Coulomb electrostatic interaction between two atoms with a soft core.
+The potential energy is defined as
+```math
+V(r_{ij}) = \frac{q_i q_j}{4 \pi \varepsilon_0 (r_{ij}^6 + \alpha * sigma_{ij}^6 * \lambda^p)^{\frac{1}{6}}}
+```
+
+Here, ``\\alpha``, ``\\lambda``, and ``\\p`` are `sc_softness`, `sc_lambda`, and `sc_power` respectively which are used
```

Comment:
Given that we generally use Greek characters elsewhere for keyword arguments I think these can be `α`, `λ` and `p`.